app/assets/stylesheets/tufts/show_actions.scss

```diff
@@ -0,0 +1,37 @@
+.show-actions {
+  margin:0;
+  padding:0;
+  max-width: 52%; 
+  padding-top: 1em; 
+  padding-bottom: 1em;
+  margin-bottom: 1em;
+  h4 {
+    padding: 0; 
+    margin: 0;
+    margin-left: .5em;
+    margin-bottom: 1em;  
+  }
+}
+
+.show-action-buttons {
+  list-style: none;
+  margin:0;
+  padding: 0;
+  li {
+  list-style:none;
+  margin-bottom:.5em;
+  margin-left:.5em;
+    a {
+        max-width: inherit;
+        padding: .7em;
+        width: 100%;
+        min-height:25px;
+    }
+    i {
+        float:left;
+        margin-left: 10px;
+        margin-right: 10px;
+        margin-top: 2px;
+    }
+  }
+}
```

Comment:
You should add ending newline and trailing whitespace configuration to your editor, @little9. :nerd_face:

Reply:
@no-reply should I do this? https://www.gnu.org/software/emacs/manual/html_node/emacs/Customize-Save.html ?  `(setq require-final-newline t)`